app/views/suggested_edit/show.html.erb

```diff
@@ -5,7 +5,7 @@
 
 <h1>Review Suggested Edit</h1>
 
-<% if params[:show_decided] != '1' && !current_user&.privilege?('edit_posts') %>
+<% if params[:show_decided] != '1' && !current_user&.privilege?('edit_posts') && !may_decide %>

```

Comment:
Can't we simply replace `!current_user&.privilege?('edit_posts')` with `!may_decide` as it does the same thing?